tests/unit/loadsave.tcl

```diff
@@ -0,0 +1,50 @@
+set server_path [tmpdir "server.rdb-encoding-test"]
+set testmodule [file normalize tests/modules/datatype.so]
```

Comment:
This won't get built in normal unit tests... Can you verify this works from a pristine repo with just "make test".